modules/logger/src/logger.c

```diff
@@ -45,7 +45,6 @@ static int addJSONString(FILE* fout, const char* jsonString)
         }
         else
         {
-            LogInfo("Message Received to Log: %s", jsonString);
```

Comment:
@damonbarry , if you check this line was removed. As the tittle on this commit say.